### PR TITLE
Opam+dune+ocamlformat

### PR DESCRIPTION
**Pull request by dinosaure**

### CODE REVIEW

File: src/b64.ml [hunk 1]
```diff
@@ -41,52 +44,50 @@ let decode ?alphabet input =
...
-  with
-    Not_found -> None
+  try Some (decode ?alphabet input) with Not_found -> None
 
-let encode ?(pad=true) ?alphabet input =
+let encode ?(pad = true) ?alphabet input =
   let length = String.length input in
-  let words = (length + 2) / 3 in (* rounded up *)
+  let words = (length + 2) / 3 in
+  (* rounded up *)
```
Comment (hannesm) [inline src/b64.ml]: i disagree with this ocamlformat change, is there a way to configure it to not insert the newline?

**Comment by hannesm** [comment]
you'll need to adjust the `opam-version` (first line of the opam file) to be `2.0` now.

**Comment by dinosaure** [comment]
`opam lint` is green 👍 !

File: src/b64.ml (hunk 1 above)
Comment (dinosaure) [reply src/b64.ml]: I did not find something about that in `ocamlformat` ... Should be an issue (@gpetiot?).

Comment (gpetiot) [reply src/b64.ml]: You can place the comment just before the `in` keyword.

**Comment by dinosaure** [comment]
Ready to merge.

**Comment by dinosaure** [comment]
Ok, it's just an usual update to last version of `dune` and last version of `opam`, I will merge if nobody have a strong opinion against this PR.

**Comment by avsm** [comment]
also fixed the makefile in 4f7366bab0167bf3c5e45f6f6be6eb80ea6b5115

**Pull request merged**